mlflow/server/js/src/components/ExperimentView.js

```diff
@@ -450,7 +450,8 @@ class ExperimentView extends Component {
     const paramKeyFilter = new KeyFilter();
     const metricKeyFilter = new KeyFilter();
     const andedExpressions = [];
-    this.props.onSearch(paramKeyFilter, metricKeyFilter, andedExpressions, "");
+    this.props.onSearch(paramKeyFilter, metricKeyFilter, andedExpressions, "",
+      LIFECYCLE_FILTER.ACTIVE);
```

Comment:
Why does an undefined lifecycle filter show only deleted? This seems as much a bug on the server.

Reply:
Good question, `this.props.onSearch` doesn't actually directly make an API request here - the logic that causes deleted runs to appear is actually in the JS: https://github.com/mlflow/mlflow/blob/master/mlflow/server/js/src/components/ExperimentPage.js#L116 (if `lifecycleFilter` is undefined, we fall through to searching for deleted runs). I'll go ahead & merge this to unblock the release, but we probably should throw an error on an invalid lifecycle stage in that method